packages/airnode-examples/integrations/coingecko-cross-chain-authorizer/create-config.ts

```diff
@@ -0,0 +1,236 @@
+import { Config } from '@api3/airnode-node';
+import {
+  createCloudProviderConfiguration,
+  createNodeVersion,
+  generateConfigFile,
+  getAirnodeRrpAddress,
+  getChainId,
+} from '../config-utils';
+
+const createConfig = async (generateExampleFile: boolean): Promise<Config> => ({
+  chains: [
+    {
+      maxConcurrency: 100,
+      authorizers: {
+        requesterEndpointAuthorizers: ['0xE2E0000000000000000000000000000000000000'],
```

Comment:
Nit: You can put them into secrets for easier implementation. But I know they are not secrets in practice so hardcoding makes sense as well.

Reply:
Thanks- I considered this, but because canonically we do not store addresses like this as secrets I opted instead for the placeholder addresses that get replaced.